src/app/pages/LandingPage/components/Promotions/components/PromotionCard/index.tsx

```diff
@@ -79,20 +73,28 @@ export const PromotionCard: React.FC<IPromotionCardProps> = ({
             <div className="tw-w-24">
               <div className="tw-flex tw-items-center">
                 <div className="tw-z-10">
-                  <AssetLogo src={AssetsDictionary.get(logoAsset1).logoSvg} />
+                  <img
+                    className={styles.assetLogo}
+                    src={AssetsDictionary.get(logoAsset1).logoSvg}
+                    alt="asset 1"
+                  />
                 </div>
                 {logoAsset2 && (
                   <div className="tw--ml-6">
-                    <AssetLogo src={AssetsDictionary.get(logoAsset2).logoSvg} />
+                    <img
+                      className={styles.assetLogo}
+                      src={AssetsDictionary.get(logoAsset2).logoSvg}
+                      alt="asset 2"
+                    />
```

Comment:
I'v seen we have the same kind of showing two assets like this.
Maybe it's good to move them into a component with sizing and reuse it.
<img width="176" alt="Screen Shot 2022-02-18 at 6 11 14 PM" src="https://user-images.githubusercontent.com/81699526/154703766-8b0ce687-5942-4f89-8c85-ea6aa1a9cd83.png">